.github/workflows/release.yml

```
@@ -144,17 +144,24 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v3
 
+      - name: Check repository owner
+        id: check-repo-owner
+        uses: ./.github/actions/check-repo-owner
+
       - name: Setup
+        if: ${{ steps.check-repo-owner.outputs.ngxs == 'true' }}
```

Just wondering if it is worth adding the composite action for this, or if we could just have the following instead:
```suggestion
        if: ${{ github.repository_owner == 'ngxs' }}
```

---

PS. We could also add this `if` at the job level instead.
This example from the docs actually does almost the same as what you are doing here:
https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution#example-only-run-job-for-specific-repository